Orange/widgets/cpewidget.py

```diff
@@ -79,42 +93,49 @@ def property_changed(self, name):
-                # handle missing commas in MATLAB-style list literals
-                if " " in content:
-                    if "'" in content:
-                        pass  # assume that these are quoted strings, ignore
-                    elif "," not in content:
-                        content = ", ".join(content.split())
-                # fix up specific bracket type
-                if content[0] in openchars and (content[0] != openchar):
-                    content = openchar + content[1:]
-                if content[-1] in closechars and (content[-1] != closechar):
-                    content = content[:-1] + closechar
+                # parse list default value and return
+                if content in NoneUIValue:
+                    content = 'None'
+                    setNoneUIValue = True
+                else:
+                    # handle empty entry
+                    if not content:
+                        content = "[]"
+                    # handle missing brackets
+                    if content[0] not in openchars:
+                        content = "[" + content
+                    if content[-1] not in closechars:
+                        content = content + "]"
+                    # strip whitespace between brackets and list
+                    content = content[0] + content[1:-1].strip() + content[-1]
+                    # replace semicolons by commas and warn
+                    if ";" in content:
+                        content.replace(";", ",")
+                        warn_once("Semicolons are not a valid character in "
+                                  "NeuroPype lists (replacing by commas).")
+                    # handle missing commas in MATLAB-style list literals
+                    if " " in content:
+                        if "'" in content:
+                            pass  # assume that these are quoted strings, ignore
+                        elif "," not in content:
+                            content = ", ".join(content.split())
+                    # fix up specific bracket type
+                    if content[0] in openchars and (content[0] != openchar):
+                        content = openchar + content[1:]
+                    if content[-1] in closechars and (content[-1] != closechar):
+                        content = content[:-1] + closechar
                 try:
                     # attempt to evaluate as expression
                     value = eval(content, None, np.__dict__)
                 except Exception:
-                    raise RuntimeError("Incorrectly formatted list: use "
-                                       "[value, value, ...] as format.")
+                    raise RuntimeError("Incorrectly formatted list: use [value, value, ...] as format.")
             else:
                 # Evaluate string as pure Python code.
                 content = getattr(self, name)
                 try:
+                    if content in NoneUIValue:
```

Comment:
I'd move this one line up out of the try (since the try is really on there because the eval() might throw), that makes the purpose of the try more clear (otherwise one might wonder if content in NoneUIValue might throw or something like that)